### PR TITLE
Ewl 839 create player statistics

### PR DESCRIPTION
**Pull request by degouville**
# About the type of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Test plan

- [ ] Empty data
- [ ] Invalid data
- [x] Correct data

# Checklist
### Before code review

- [x] Move Jira ticket into PR
- [x] Check with another developers that features sounds good

### CODE REVIEW

File: src/modules/PlayerStatistics/utils.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export const setGradient = ({ firstValue, secondValue, maximumValue, colorOne, colorTwo }) => {
```
Comment (JulienKode) [inline src/modules/PlayerStatistics/utils.js]: Should we have a dedicated test file for this utils ?

File: src/modules/PlayerStatistics/utils.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  // NOTE: share values on a range of 100%
+  const b = Math.floor(((firstValue / maximumValue) * 100) / 2);
```
Comment (saad-kh) [inline src/modules/PlayerStatistics/utils.js]: is it okay if a value goes beyond 50%?

File: src/modules/PlayerStatistics/index.test.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { ThemeProvider } from 'emotion-theming';
+import theme from '../../theme';
```
Comment (JulienKode) [inline src/modules/PlayerStatistics/index.test.js]: Absolute import

File: src/modules/PlayerStatistics/utils.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  const c = Math.floor(((secondValue / maximumValue) * 100) / 2);
+  const a = 100 / 2 - b;
+
+  const startAt = 0;
+  const offset = startAt + a;
```
Comment (saad-kh) [inline src/modules/PlayerStatistics/utils.js]: Why do we need to define startAt?

File: src/modules/PlayerStatistics/index.test.js (hunk 3 above)
Comment (JulienKode) [reply src/modules/PlayerStatistics/index.test.js]: Like `import theme from 'src/theme';`

File: src/modules/PlayerStatistics/utils.js (hunk 1 above)
Comment (degouville) [reply src/modules/PlayerStatistics/utils.js]: I think we should... maybe with some JSDoc blocks

File: src/modules/PlayerStatistics/Comparison.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+      <p>{addPercentMark(isPercent, one)}</p>
+      <p>{title}</p>
+      <p>{addPercentMark(isPercent, two)}</p>
+    </StyledComparison>
+  );
+};
+
+Comparison.propTypes = {
+  title: PropTypes.string.isRequired,
+  currentValues: PropTypes.shape(PropTypes.object).isRequired,
```
[the remark this answers is not in the document]
Comment (JulienKode) [reply src/modules/PlayerStatistics/Comparison.js]: Yes and declare all values

File: src/modules/PlayerStatistics/utils.js (hunk 1 above)
Comment (JulienKode) [reply src/modules/PlayerStatistics/utils.js]: @degouville good idea 😊

File: src/modules/PlayerStatistics/index.test.js (hunk 3 above)
Comment (degouville) [reply src/modules/PlayerStatistics/index.test.js]: of course

Comment (degouville) [reply src/modules/PlayerStatistics/index.test.js]: done ✅

File: src/modules/PlayerStatistics/index.test.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+  const wrapper = shallow(
+    <ThemeProvider theme={theme}>
+      <PlayerStatistics names={names} comparisonPoints={comparisonPoints} />
+    </ThemeProvider>
+  );
+
+  const playerStatistics = wrapper.find(PlayerStatistics);
+
+  // Expect
+  expect(playerStatistics).toMatchSnapshot();
```
[the remark this answers is not in the document]
Comment (JulienKode) [reply src/modules/PlayerStatistics/index.test.js]: Here use wrapper with the theme so each time we will change the theme we will have to update the snapshot

[the remark this answers is not in the document]
Comment (JulienKode) [reply src/modules/PlayerStatistics/index.test.js]: https://github.com/EurosportDigital/web-toolkit/pull/294

[the remark this answers is not in the document]
Comment (JulienKode) [reply src/modules/PlayerStatistics/index.test.js]: we have that problem here: https://github.com/EurosportDigital/web-toolkit/pull/292#issuecomment-516786741

[the remark this answers is not in the document]
Comment (saad-kh) [reply src/modules/PlayerStatistics/index.test.js]: ah okay

File: src/modules/PlayerStatistics/Comparison.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (degouville) [reply src/modules/PlayerStatistics/Comparison.js]: yes I'll add it

File: src/modules/PlayerStatistics/utils.js (hunk 2 above)
Comment (degouville) [reply src/modules/PlayerStatistics/utils.js]: Not as firstValue should not be greater than maximumValue

**Pull request merged**